modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -2543,7 +2538,7 @@ def sort_function(df):  # pragma: no cover
             )
 
         result = self._apply_func_to_range_partitioning(
-            key_column=columns[0], func=sort_function, ascending=ascending, **kwargs
+            key_columns=columns[0], func=sort_function, ascending=ascending, **kwargs
```

Comment:
we still pass only one column in case of `.sort_values()` as its performance with new functionality has not been yet tested